index.html

```diff
@@ -0,0 +1,289 @@
+                            <button>
+                                Buy this bundle
+                            </button>
+                        </div>
+                        <div class="info">
+                            <ul style="list-style-type: circle;">
+                                <div class="list">
+                                    <div class="tick">
+                                        <img src="Icon.svg" alt="svg image">
+                                    </div>
+                                    <p>
+                                        Per credit : <span>$0.4</span>
+                                    </p>
+                                </div>
+                                <div class="list">
+                                    <div class="tick">
+                                        <img src="Icon.svg" alt="svg image">
+                                    </div>
+                                    <p>
+                                        Save: <span>$3,000</span>
+                                    </p>
+                                </div>
+                                <div class="list">
+                                    <div class="tick">
+                                        <img src="Icon.svg" alt="svg image">
+                                    </div>
+                                    <p>
+                                        Used for up to <span>12,000 entities</span>
+                                    </p>
+                                </div>
+                                <div class="list">
+                                    <div class="tick">
+                                        <img src="Icon.svg" alt="svg image">
+                                    </div>
+                                    <p>
+                                        Assign credit to users
+                                    </p>
+                                </div>
+                                <div class="list">
+                                    <div class="tick">
+                                        <img src="Icon.svg" alt="svg image">
+                                    </div>
+                                    <p>
+                                        Preview before buying
+                                    </p>
+                                </div>
+                                <div class="list">
+                                    <div class="tick">
+                                        <img src="Icon.svg" alt="svg image">
+                                    </div>
+                                    <p>
+                                        Use within 30 days
+                                    </p>
+                                </div>
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="All_users">
```

Comment:
add overflow scroll, fix the width of table to a minimum of e.g 900px so table elements layout remains readable